tests/baselines/reference/jsdocFunctionTypeFalsePositive.types

```diff
@@ -1,6 +1,6 @@
 === /a.js ===
 /** @param {<} x */
 function f(x) {}
->f : (x: any) => void

```

Comment:
IDK if this is bad or not; this is clearly malformed JSDoc but before we just got any. Now we think we're starting type parameters (which is normal for parsing a type) and then assume it's probably a function signature which is the only thing that can start with `<` like this.

Reply:
Looks fine to me.